Integrate Upstream Changes

rustydb opened the pull request:
This pulls in the latest bugfix and removes our CODEOWNERS so we can make upstream PRs.

I have an open upstream PR dependent on this change: https://github.com/HewlettPackard/python-redfish-utility/pull/52

rustydb commented:
Merging this once the new build works with the `ilorest_chif.so` file.

heemstra reviewed (approved):
The consecutive `sh` calls should eventually be in a script that is called instead.

rajeevkallur committed:
version 3.5.1

Commit:
Merge remote-tracking branch 'HPE/master' into integration

Commit:
Fixup pyproject.toml end of file

Commit:
Update setup.py version

This has to be updated in-tandem with src/versioning.py until this
entire module is converted to a newer layout. Once converted,
versioning.py could obtain the version from setup.py from
`pkg_resources.get_distribution('ilorest').version`.

Commit:
Update changelog ; Cray builds key off of this
